python/hsfs/core/statistics_api.py

```diff
@@ -0,0 +1,46 @@
+#
+#   Copyright 2020 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from hsfs import client
+
+
+class StatisticsApi:
+    def __init__(self, feature_store_id, entity_type):
+        """Statistics endpoint for `trainingdatasets` and `featuregroups` resource.
+
+        :param feature_store_id: id of the respective featurestore
+        :type feature_store_id: int
+        :param entity_type: "trainingdatasets" or "featuregroups"
+        :type entity_type: str
+        """
+        self._feature_store_id = feature_store_id
+        self._entity_type = entity_type
+
+    def post(self, metadata_instance, statistics):
```

Comment:
I think we should also allow users to get the statistics from the API.

Reply:
done

Reply:
done